osu.Game/Graphics/UserInterface/ShowMoreButton.cs

```diff
@@ -1,30 +1,36 @@
-// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
```

Comment:
What's happened here? This looks like a newline change, I'd double-check your `git` settings.